addons/website/static/src/builder/plugins/options/avatars_option_plugin.js

```diff
@@ -0,0 +1,97 @@
+import { Plugin } from "@html_editor/plugin";
+import { registry } from "@web/core/registry";
+import { BuilderAction } from "@html_builder/core/builder_action";
+import { renderToElement } from "@web/core/utils/render";
+import { AvatarsHeaderMiddleButtons } from "./avatars_header_buttons";
+
+function _updateZIndex(el) {
+    const avatars = el.querySelectorAll(".o_avatar");
+    if (!el.classList.contains("o_first_on_top")) {
+        avatars.forEach(avatar => {
+            avatar.style.setProperty("--avatar-z-index", "");
+        });
+        return;
+    }
+    const totalAvatars = avatars.length;
+    avatars.forEach((avatar, index) => {
```

Comment:
`avatarEls.forEach((avatarEl, index) => {`

@chgo-odoo I missed it in my first review 😉